README.md

```diff
@@ -65,6 +65,16 @@ Please file issues before filing PRs. For PRs to be accepted, contributors must
 
 Reviewers, please ensure that the CLA has been signed by referring to [the contributors tool](https://contributors.corp.mongodb.com/) (internal link).
 
+## Linting
+
+This project uses the following linters upon every Pull Request:
+
+* `Black` is a tool that verifies if Python code is properly formatted
```

Comment:
Do we need to add `gofumpt` here as well since we add `Black` ?

Reply:
I haven't added it to MCO yet

Reply:
Ok, since this one has slightly more priority, I'll go ahead and merge it. If we introduce `gofumpt`, please update this section as well.

Reply:
Oh but there is no PR opened for gofumpt on MCO anyway, feel free to merge
I will put that on my list but as low priority